mindmeld/components/dialogue.py

```diff
@@ -1011,6 +1013,56 @@ async def call_async(self, request, responder):
         """
         self(request, responder)
 
+    def _check_invoke_complete(self, request, responder):
+        """Check if all entities captured for invoke, return control to handler"""
+        _captured = []
+        _reqd = []
+        if request.entities:
+            for e in request.entities:
+                _captured.append(e['type'])
+
+        for e in self._entity_form:
+            _reqd.append(e.entity)
+
+        # check if already filled in previous turn, reset call back to original
+        if sorted(_captured) == sorted(_reqd):
```

Comment:
If the order of the lists doesn't matter, could just use sets.

Reply:
Not needed anymore, removed this as well.